src/Resources/views/Profiler/pheanstalk.html.twig

```diff
@@ -1,13 +1,14 @@
 {% extends 'WebProfilerBundle:Profiler:layout.html.twig' %}
 
 {% block toolbar %}
-    <div class="sf-toolbar-block">
-        <div class="sf-toolbar-icon">
-            <a href="{{ path('_profiler', { 'token': token, 'panel': name }) }}">
-                <span style="width:0px; height: 28px; vertical-align: middle;"></span>
-                <span class="sf-toolbar-status">{{ collector.jobCount }}</span> Jobs
-            </a>
-        </div>
+    <div class="sf-toolbar-block sf-toolbar-block-leezy-pheanstalk sf-toolbar-status-normal">
+        <a href="{{ path('_profiler', { 'token': token, 'panel': name }) }}">
+            <div class="sf-toolbar-icon">
+                {{ include('@LeezyPheanstalk/Profiler/database.svg') }}
+                <span class="sf-toolbar-value">{{ collector.jobCount }}</span>
+                <span class="sf-toolbar-label">Jobs</span>
+            </div>
+        </a>
         <div class="sf-toolbar-info">
             <div class="sf-toolbar-info-piece">
                 <b>Pheanstalks</b>
```

Comment:
gasp; sorry for mentioned; the page was opened yesterday and I didn't refresh :p